Dockerfile

```diff
@@ -7,6 +7,10 @@ ARG CONCURRENCY=4
 # setup the ondemand repositories
 RUN dnf -y install https://yum.osc.edu/ondemand/latest/ondemand-release-web-latest-1-6.noarch.rpm
 
+# setup yarn repository
+RUN curl --silent --location https://dl.yarnpkg.com/rpm/yarn.repo | tee /etc/yum.repos.d/yarn.repo && \
```

Comment:
Is there a way to do this without adding a dependency on `yarn` and extra repos?  One of the major benefits thus far with OnDemand has been the ability to rely on just OnDemand repo and then OS repos.  Adding additional repos makes the deployment more complicated.

Reply:
I talked with Eric offline and he came up with another solution to avoid adding extra repos, we'll use a similar process for setup as the `shell` app: https://github.com/OSC/ondemand/blob/5cb2f53c20f19805939dfc24b3d9b1d641bad288/apps/shell/bin/setup

Yarn will be installed from NPM.

Reply:
Resolved in #862